artscii/src/services/giphyService.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+
+const GIPHY_API_KEY = `7bcbIq6rDmXAuPqfDLaa1TrL7dN0q47w`;
```

Comment:
This is something we don't want committed in the repository. We should set it to an environment variable.
const GIPHY_API_KEY = process.env.REACT_APP_GIPHY_KEY
[https://create-react-app.dev/docs/adding-custom-environment-variables/](https://create-react-app.dev/docs/adding-custom-environment-variables/)

Reply:
I will do that in the subsequent PR. Since I am facing some issues implementing it, I will raise another PR with that work. Is it fine to merge this for the time being?

Reply:
I would also remove the call in the App.js file - right now it is calling the API every time the page loads. We also need the function to take in a string as an argument that it uses in the call.

Reply:
I am working on the review comments and will update PR accordingly.